src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -14372,24 +14363,17 @@ static void Test2(object x, object y)
             var compilation = CreateCompilationWithMscorlib(text,
                                                             options: TestOptions.ReleaseExe,
                                                             parseOptions: TestOptions.Regular);
-
             compilation.VerifyDiagnostics(
-                // (7,21): error CS1525: Invalid expression term 'int'
+                // (7,21): error CS1615: Argument 1 may not be passed with the 'out' keyword
                 //         Test2(x[out int x1], x1);
-                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "int").WithArguments("int").WithLocation(7, 21),
-                // (7,25): error CS1003: Syntax error, ',' expected
+                Diagnostic(ErrorCode.ERR_BadArgExtraRef, "int x1").WithArguments("1", "out").WithLocation(7, 21),
+                // (7,21): error CS0165: Use of unassigned local variable 'x1'
                 //         Test2(x[out int x1], x1);
-                Diagnostic(ErrorCode.ERR_SyntaxError, "x1").WithArguments(",", "").WithLocation(7, 25),
-                // (7,25): error CS0103: The name 'x1' does not exist in the current context
-                //         Test2(x[out int x1], x1);
-                Diagnostic(ErrorCode.ERR_NameNotInContext, "x1").WithArguments("x1").WithLocation(7, 25),
-                // (7,30): error CS0103: The name 'x1' does not exist in the current context
-                //         Test2(x[out int x1], x1);
-                Diagnostic(ErrorCode.ERR_NameNotInContext, "x1").WithArguments("x1").WithLocation(7, 30)
+                Diagnostic(ErrorCode.ERR_UseDefViolation, "int x1").WithArguments("x1").WithLocation(7, 21)
                 );
 
             var tree = compilation.SyntaxTrees.Single();
-            Assert.False(GetOutVarDeclarations(tree, "x1").Any());
+            Assert.Equal(1, GetOutVarDeclarations(tree, "x1").Count());
```

Comment:
>Assert.Equal(1, GetOutVarDeclarations(tree, "x1").Count()); [](start = 12, length = 59)

Please replace this Assert with code that verifies semantic model behavior for the declaration.